docs/docs/03-main-concepts.md

```diff
@@ -82,3 +81,26 @@ Snowpack ships with official support for [webpack](https://www.npmjs.com/package
   "plugins": [["@snowpack/plugin-webpack", {/* ... */}]]
 }
 ```
+
+#### Unbundled optimization (experimental)
+
+To try Snowpack’s unbundled optimization including JS minification and JS [module preloading](https://developers.google.com/web/updates/2017/12/modulepreload), check out `@snowpack/plugin-optimize`. The goal of this plugin is to optimize an unbundled application for production, and provide better configuration all in one place. Note that this is experimental, so the API/plugin options may change slightly as features stabilize.
+
+_Note: this replaces `buildOptions.minify`._
```

Comment:
Added a note here about deprecating `buildOptions.minify`, which we’d have to do by function of removing minification from core Snowpack behavior. Though this is a change to default behavior, I think it’s not too disruptive.

Reply:
+1. Maybe clarify as saying "this plugin"